dev/io.openliberty.jakarta.validation.v31_fat/test-applications/val31/src/val31/web/ValidationOrder.java

```diff
@@ -0,0 +1,21 @@
+/*******************************************************************************
+ * Copyright (c) 2024 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+package val31.web;
+
+import jakarta.validation.GroupSequence;
+
+/**
+ * Define a GroupSequence to specify the order of validation groups.
+ */
+@GroupSequence({ BasicInfo.class, AdvanceInfo.class })
```

Comment:
These should probably be renamed, since they're directly from the example. Something like First.class and Second.class would work, while making the ordering explicit

Reply:
Updated